### ISSUE TITLE
Prerequisites for compiling

### ISSUE THREAD

**Issue by hitam4450** [compiling-nextcloud/compiling-nextcloud#1]
Hi,
for newbies compiling source, it should be mentioned what kind of prerequisites (packages) should be present to successfully compile the source!

Regards, hitam4450

**Comment by Nebucatnetzer** [comment]
I agree that's something that should be in each guide.
Do you mean for the Ubuntu guide or in general?
I know that this packages should be present in a Debian based distro:
`binutils debootstrap fakeroot devscripts`

**Comment by hitam4450** [comment]
For each relevant guide.........

**Comment by Nebucatnetzer** [comment]
Okay I'll add it to the Ubuntu 16.04 guide. Thanks for the input.

**Comment by hitam4450** [comment]
Perhaps for DEBIAN / Jessie as well.....many thanks!

**Comment by Nebucatnetzer** [comment]
Absolutely, unfortunately I don't know how to compile it for Debian Jessie.
For me it always fails at some point. I hope someone will contribute a guide for this.
